site/content/blogs/release-v118.en.md

```diff
@@ -22,8 +22,69 @@ enabling you to focus on developing your application, instead of writing deploym
 See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## Certificate import
+
 _Contributed by [Penghao He](https://github.com/iamhopaul123/)_
 
+Developers that have domains not managed by Route 53 or want to use HTTPS for Copilot applications without any domain specified can now use a new `--import-cert-arns` flag to import any validated certificates when they create an environment, so as to use any domain/SAN specified in their imported certificates for their services deployed to the environment:
```

Comment:
```suggestion
Developers that have domains managed outside of Route 53 or who want to use HTTPS for Copilot applications without any domain specified can now use a new `--import-cert-arns` flag to import validated certificates when they create an environment. so as to use any domain/SAN specified in their imported certificates for their services deployed to the environment:
```

i'm not quite understanding the sentence/section starting with "so as to use" - is it supposed to be introducing the example below?